src/vma/dev/ring_simple.h

```diff
@@ -90,6 +90,7 @@ class ring_simple : public ring
 	friend class cq_mgr;
 	friend class cq_mgr_mlx5;
 	friend class qp_mgr;
+	friend class qp_mgr_eth_mlx5;
```

Comment:
This comment is for line 117.
`get_lkey()` is not used anywhere in the code (it's commented out). On the other hand, there are places where the ring's `m_tx_lkey` is accessed.
Please check if `get_lkey()` should be kept and used or removed.